packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts

```diff
@@ -355,6 +365,10 @@ describe("SharedTree benchmarks", () => {
 						const provider = new TestTreeProviderLite(1, factory);
 						// TODO: specify a schema for these trees.
 						const [tree] = provider.trees;
+
+						updateSchema(tree, schemaGeneralized);
+						assertSchema(tree, schemaGeneralized);
```

Comment:
I am not familiar with how these tests work exactly but I'm wondering if schema updates and assertions should occur after the inserts on line 373?